DayrSuryan/DSEthiop11.xml

```diff
@@ -489,7 +496,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   <bindingDesc>
                      <binding xml:id="binding">
                         <decoNote xml:id="b1">Non-Ethiopian binding, cardboard, reddish leather, endbands. One imprinted border, one imprinted
-                        white cross in the centre on each board, يوسف نجده الحبشي imprinted on the spine.</decoNote>
+                        white cross in the centre on each board, the name of the owner
+                        يوسف نجده الحبشي is imprinted on the spine.</decoNote>
```

Comment:
The name should be marked up with `<foreign xml:lang="ar">`

Reply:
Done